source/content/supported-drupal.md

```diff
@@ -2,28 +2,38 @@
 title: Supported Drupal Versions
 description: Learn which versions of Drupal are currently supported, as well as additional compatibility information.
 tags: [libraries, updates]
-contributors: [wordsmither]
+contributors: [wordsmither, scottbuscemi]
 contenttype: [doc]
 innav: [true]
 categories: [cms]
 cms: [drupal]
 audience: [development]
 product: [--]
 integration: [--]
-reviewed: "2024-11-04"
+reviewed: "2025-01-14"
 ---
 
 The following table indicates availability of the specified Drupal versions, as well as our usage recommendations and our support scope.
 
 | Drupal version | Available | Recommended | Supported |
 | ----------- | :---------: | :---------: | :---------: |
+| CMS          | <span style="color:green">✔</span> | <span style="color:green">✔</span>           | <span style="color:green">✔</span>
```

Comment:
is it redundant to change this to `Drupal CMS` here?

Reply:
I see the idea but we don't put "Drupal" in front of all the other versions.

Reply:
I could link to https://new.drupal.org/drupal-cms in the table to help maybe?

Reply:
I agree `Drupal CMS` feels right